### PR TITLE
Bump user-identification-manager to rev with smart card id source

### PR DESCRIPTION
**Pull request by martin-ejdestig**

### CODE REVIEW

File: recipes-extended/user-identification-manager/user-identification-manager_git.bb [hunk 1]
```diff
@@ -11,14 +11,22 @@ PV = "0.1.0+git${SRCPV}"
...
-SRCREV = "4ada21266f72b4e98337dfc787b2906427f7ed20"
+SRCREV = "5bcddca8ebdf546d8e86421f71b2412da8146c64"
 
 inherit meson systemd
 
 DEPENDS += "glibmm gdbus-codegen-glibmm-native"
 
 S = "${WORKDIR}/git"
 
+PACKAGECONFIG ??= " \
```
Comment (sashko) [inline recipes-extended/user-identification-manager/user-identification-manager_git.bb]: it cannot be set anywhere else, so you can use `=` instead, but whatever of course

File: recipes-extended/user-identification-manager/user-identification-manager_git.bb [hunk 2]
```diff
@@ -11,14 +11,22 @@ PV = "0.1.0+git${SRCPV}"
 PR = "r0"
```
Comment (sashko) [inline recipes-extended/user-identification-manager/user-identification-manager_git.bb]: PV and PR should either be used or removed

File: recipes-extended/user-identification-manager/user-identification-manager_git.bb (hunk 1 above)
Comment (martin-ejdestig) [reply recipes-extended/user-identification-manager/user-identification-manager_git.bb]: A weak default value is what they use in the mega manual documentation for PACKAGECONFIG and is most commonly used in poky/meta-oe, so I did the same.

File: recipes-extended/user-identification-manager/user-identification-manager_git.bb (hunk 2 above)
Comment (martin-ejdestig) [reply recipes-extended/user-identification-manager/user-identification-manager_git.bb]: Not relevant for this pr. I remember this being discussed at length in some other pr and do not want to dig into it now.

**Pull request merged**